pyproject.toml

```diff
@@ -16,14 +16,10 @@ homepage = "https://github.com/sparrowml/sparrow-tracky"
 # Keywords description https://python-poetry.org/docs/pyproject/#keywords
 keywords = []  #! Update me
 
-[[tool.poetry.source]]
-url = "https://pypi.fury.io/sparrow/"
-name = "sparrow"
-
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 scipy = "^1.8"
-sparrow-datums = ">=0.7.0"
+sparrow-datums = ">=0.8.1.dev"
```

Comment:
TODO: need to switch this back to `0.8.1` after merging sparrowml/sparrow-datums#35

Reply:
Since I was working with a new repo today "cowhead-detector" I made an effort to look through the pyproject file. It's nice to see the consistent structure across the various projects, and each repetition helps.